pkg/controller/operators/olm/config.go

```diff
@@ -112,6 +115,18 @@ func WithLogger(logger *logrus.Logger) OperatorOption {
 	}
 }
 
+func WithProtectedCopiedCSVNamespaces(namespaces string) OperatorOption {
+	return func(config *operatorConfig) {
+		if namespaces == "" {
+			return
+		}
+
+		for _, ns := range strings.Split(namespaces, ",") {
```

Comment:
nit: Should this just take a slice of strings that it uses instead of needing to split a single string?

Reply:
Good question.

The flag, which is used to configure OLM, is passed in as a string, so I opted to convert that string into an array within the function.

Reply:
I still believe that it makes sense to live here.